aiofirebase/__init__.py

```diff
@@ -83,11 +83,13 @@ def __init__(self, base_url, auth=None, loop=None):
             elif key == 'data':
                 await callback(event=event, data=json.loads(value))
 
-    async def _request(self, *, method, value=None, path=None, params=None):
+    async def _request(self, *, method, value=None, path=None, params={}):
         """Perform a request to Firebase."""
         url = posixpath.join(self._base_url, path.strip('/')) if path else self._base_url
         url += '.json'
         data = json.dumps(value) if value else None
+        if self._auth:
```

Comment:
Because I allow any arbitrary params to be given in each call, a user may override the auth set when creating the instance.